package.json

```diff
@@ -46,7 +46,9 @@
     "format:js": "npm run format:provided '**/*.js' '**/*.ts' '**/*.tsx'",
     "format:css": "npm run format:provided '**/*.scss' '**/*.css'",
     "format:provided": "prettier --write",
-    "setup:jurassic-tube": "./bin/jurassic-tube-setup.sh"
+    "setup:jurassic-tube": "./bin/jurassic-tube-setup.sh",
+    "start:jurassic-tube": "./docker/bin/jt/tunnel.sh",
+    "stop:jurassic-tube": "./docker/bin/jt/tunnel.sh break"
```

Comment:
nit: personally I think it makes sense, now that there are more than 1 script for jurassic tube, to have `jurassic-tube:` as prefix, ie
`jurassic-tube:setup`
`jurassic-tube:start`
`jurassic-tube:stop`
but that is just personal preference.

What do you think @dmallory42 ?

Reply:
I agree @shendy-a8c. This is something I realised after I did the initial Jurassic Tube setup script, so I'm really glad that you mentioned it! I think it is more consistent with the other commands we have.

Reply:
I think we all are aligned on this. Just a thought, can we keep the prefix as `tube` instead of the whole `jurassic-tube`.

`tube:setup`
`tube:start`
`tube:stop`

easier to type, also consistent with the [a8c-wp-env](https://github.com/Automattic/a8c-wp-env/blob/trunk/package.json) repo.

Reply:
Sounds good to me. Please also try to find where `setup:jurassic-tube` has been mentioned (in P2s, FG, readmes, etc).

Reply:
Thankfully (depending on your perspective 😆), I don't _think_ that `setup:jurassic-tube` was very well documented yet. When we originally created the script, there was a conversation about where documentation should go because we didn't want to advertise it publicly on the repo (since it's only intended for Automatticans to use).

So, this might be a good opportunity to also document these commands in an appropriate place - either a P2 post or as part of the WCPay Setup pages on Field Guide.

What do you think @shendy-a8c @naman03malhotra? :)

Reply:
Script documentation discussed here: https://github.com/Automattic/woocommerce-payments/pull/1578#discussion_r614756681

Reply:
Made changes in commit https://github.com/Automattic/woocommerce-payments/pull/1578/commits/23abb577033b97e8824c195817fdf8a980dfa2f0